standard-controller/src/main/java/io/enmasse/controller/standard/StandardController.java

```diff
@@ -124,16 +126,16 @@ public void stop() {
                     }
                 }
             } finally {
-                openShiftClient.close();
+                kubeClient.close();
                 log.info("StandardController stopped");
             }
         }
     }
 
-    private static boolean isOpenShift(NamespacedOpenShiftClient client) {
+    private static boolean isOpenShift(NamespacedKubernetesClient client) {
```

Comment:
There's a version of this method in src/main/java/io/enmasse/api/server/ApiServer.java that this PR changes into a retry loop.  If it is important to retry there, why not here too?

Reply:
I will remove the retry from the api-server because it is no longer needed

Reply:
I removed this check altogether, as the openshift-specific behavior is never used in standard-controller